cmd/agent/gui/gui.go

```diff
@@ -232,5 +233,10 @@ func parseBody(r *http.Request) (Payload, error) {
 		return p, e
 	}
 
+	// perform some validation of the input
+	if _, err := strconv.ParseInt(p.CaseID, 10, 0); err != nil {
```

Comment:
This validation would likely be a better fit to add to [the caller](https://github.com/DataDog/datadog-agent/blob/main/cmd/agent/gui/agent.go#L135) of this function instead of here.

Reply:
Oh, indeed, this function is not specific to the _flare_ payload.